jekyll/_cci2/language-android.md

```diff
@@ -9,20 +9,37 @@ order: 9
 
 *[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Android*
 
-This guide will help you get started with an Android application on CircleCI.
+This document describes
+how to set up an Android project on CircleCI
+in the following sections.
 
 * TOC
 {:toc}
 
 ## Overview
 
-There are some assumptions made in this guide:
+This guide provides an introduction to Android development on CircleCI.
+If you are looking for a `.circleci/config.yml` template for Android,
+see the [Sample Configuration](#sample-configuration) section of this document.
 
-- We assume that your Android project is built with `gradle` (this is the default for projects created with [Android Studio](https://developer.android.com/studio).
-- We assume that your project is located in the root of your `git` repository, with the application located in a subfolder named `app`.
```

Comment:
Assumptions belong in an explicit **Prerequisites** section.

Reply:
cool, agreed.